src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task() {
+        this("");
+    }
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+    public void setDone() {
+        this.isDone = true;
```

Comment:
The `this` keyword should be used when there is another variable of the same name, typically to avoid confusion in such scenarios